hooks/afterRegistration.ts

```diff
@@ -0,0 +1,45 @@
+import { currentStoreView } from '@vue-storefront/core/lib/multistore'
+import PaypalButton from '../components/Button.vue'
+import * as types from '../store/mutation-types'
+
+export function afterRegistration({ Vue, config, store, isServer }) {
+  // const VSF_PAYPAL_CODE = 'vsfpaypal'
+  const VSF_PAYPAL_CODE = 'paypal_express'
+  let correctPaymentMethod = false
+  const placeOrder = () => {
+    if (correctPaymentMethod) {
+      // Vue.prototype.$bus.$emit('checkout-do-placeOrder', {})
+    }
+  }
+
+  if (!isServer) {
+    Vue.prototype.$bus.$on('set-unique-payment-methods', methods => {
+      store.commit('payment-paypal-magento2/' + types.SET_BACKEND_PAYMENT_PAYPAL_EXPRESS, methods)
+    })
+
+    Vue.prototype.$bus.$on('checkout-before-placeOrder', placeOrder)
+
+    // Mount the info component when required
+    Vue.prototype.$bus.$on('checkout-payment-method-changed', (paymentMethodCode) => {
+      if (paymentMethodCode === VSF_PAYPAL_CODE) {
```

Comment:
@tonyisworking We can improve 
`correctPaymentMethod = paymentMethodCode === VSF_PAYPAL_CODE`